framework/utils/InputData.py

```diff
@@ -427,7 +427,7 @@ def generateLatex(cls, recDepth=0):
     # TODO is this a good idea? -> disables underscores in math mode :(
     if recDepth == 0:
       # assure underscore is escaped, but not doubly
-      msg = re.sub(r'(?<!\\)_', '\_', msg)
+      msg = re.sub(r'(?<!\\)_', 'r\_', msg)
```

Comment:
Opps, r should be outside the quote marks.

Reply:
true, sorry

Reply:
Fixed.